src/textual/widgets/_button.py

```diff
@@ -39,11 +39,6 @@ class Button(Static, can_focus=True):
         text-style: bold;
     }
 
-    Button.-disabled {
-        opacity: 0.4;
-        text-opacity: 0.7;
-    }
-
```

Comment:
I'd expect to either rewrite this rule with the new pseudoclass _or_ to define the default styling `*:disabled` under `App.DEFAULT_CSS` to use these values.
With this change plus the default CSS under `App`, buttons are now styled differently when disabled, aren't they?

Reply:
There is a very subtle difference in the disabled style of `Button`, yes. @willmcgugan ask that they be lifted ever so slightly, along with everything else that can now be disabled. That is... the `Button` style was hoisted up to `App`, applied to `*` with `:disabled`, and also tweaked ever so slightly.